### PR TITLE
Milestone #2  - Calculator UI

### PR DESCRIPTION
**Pull request by Anjacodes**
In this milestone, I:

* Added a components folder
* Created a calculator component
* Built the UI of the calculator
* Added basing styling

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container">
+      <Calculator />
```
Comment (AshakaE) [inline src/App.js]: - [ ] Please make the app.js a class based component

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Anjacodes  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Anjacodes) [reply src/App.js]: Hi @AshakaE, thank you for pointing this out 🙏 ! I have gone ahead and edited the code accordingly.

**Review by D4nielJ — APPROVED** [review]
## Hello @Anjacodes, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**